checkout-extension/shopify.extension.toml.liquid

```diff
@@ -3,7 +3,11 @@
 
 # The version of APIs your extension will receive. Learn more:
 # https://shopify.dev/docs/api/usage/versioning
-api_version = "2025-01"
+{% if flavor contains "preact" -%}
+api_version = "2025-07"
+{%- else -%}
```

Comment:
Might be one too many `-`?
```suggestion
{% else %}
```

I verified that this one works as expected: https://github.com/Shopify/extensions-templates/blob/63a1d061e8799633257d21c845a59b4068648b1c/admin-action/shopify.extension.toml.liquid#L1